pages/linux/scriptreplay.md

```diff
@@ -0,0 +1,15 @@
+# scriptreplay
+
+> Output content of session files created by script command to standard output.
+
+- Replay {{path/to/session.out}} at the speed it was recorded:
```

Comment:
```suggestion
- Replay a typescript at the speed it was recorded:
```
Let's not use `{{curly braces}}` in descriptions, only in commands. It is better to form a more understandable sentence. The same goes for all the other examples.